concepts/05 Widgets/Scheduler/030 Appointments/020 Add Appointments/010 User Interaction.md

```diff
@@ -45,4 +45,57 @@ To prevent an end user from adding an appointment, set the **editing**.[allowAdd
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            // ...
+            :editing="editing"
```

Comment:
Things that in jQuery are configured with an object, in Vue and React are configured using nested components.